imtest/imtest.c

```diff
@@ -2880,13 +2890,14 @@ int main(int argc, char **argv)
 
     /* keep this in alphabetical order */
     static const char short_options[] =
-        "?I:P:X:a:cf:hik:l:m:n:o:p:qr:st:u:vw:x:z";
+        "?I:P:X:H:a:cf:hik:l:m:n:o:p:qr:st:u:vw:x:z";
 
     static const struct option long_options[] = {
         /* n.b. -? is duplicated as -h */
         { "pidfile", required_argument, NULL, 'I' },
         { "protocol", required_argument, NULL, 'P' },
         /* n.b. -X is duplicated as -x */
+        { "haproxy-clientip", required_argument, NULL, 'H' },
         { "authname", required_argument, NULL, 'a' },
         { "do-challenge", no_argument, NULL, 'c' },
         { "input-filename", required_argument, NULL, 'f' },
```

Comment:
When you fix the ordering in short_options, fix it here too.  This struct is a hassle to read if it's not sorted by the short opt.

Reply:
Done